jenkins-pipelines/.properties

```diff
@@ -39,7 +39,7 @@ IMAGE_NAME="diabetestrained"
 #  ALLOW_RUN_CANCEL=true
 
 # Flag to allow rebuilding the AML Environment after it was built for the first time. This enables dependency updates from conda_dependencies.yaml.
-AML_REBUILD_ENVIRONMENT=false
+AML_REBUILD_ENVIRONMENT=true
```

Comment:
Would this suggest that every time we would run this, the environment should be rebuilt? Or this is just a one-time thing because we changed some dependency modules?

Reply:
Yes, I intentionally flipped the flag to true so that it rebuilt's the environment. This is because the azure-ml sdk has lot of changes being pushed every time and these seem to be breaking the code (backward incompatible changes) and so it might be better to let the package consumer decide when to turn off the flag instead.